.gitignore

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
Please, check that you have resolved conflicts very well. You can reach out to @katherine95 in case you are blocked

Reply:
Resolved